### ISSUE TITLE
Energy problem

### ISSUE THREAD

**Issue by MichaelHendriks** [ArjanKranenburg/virtual-devices#111]
Creating a button is no problem. But the energy input does not seem to work.
Status 0W

**Comment by ArjanKranenburg** [comment]
Since it is 'Virtual', it does not consume any energy. I've set the energy consumption to 0.

**Comment by ArjanKranenburg** [comment]
But I think you can overwrite that in 'Advanced settings'

**Comment by MichaelHendriks** [comment]
> Since it is 'Virtual', it does not consume any energy. I've set the energy consumption to 0.
Behind my switch device (switch with VD) is a device that uses more energy when I turn It on. Homey makes it possible to enter these consumption data.

**Comment by MichaelHendriks** [comment]
> But I think you can overwrite that in 'Advanced settings'
Nope! That's the problem

**Comment by ArjanKranenburg** [comment]
Shouldn't you monitor the real device then?
If you copy that consumption to a Virtual device, Homey may count it twice.

**Comment by MichaelHendriks** [comment]
No, I have a ventilation system with 3 positions. The 3 positions are switched by Homey with a wireless module that uses less than 3 watts. The ventilation consumes more, depending on the position. I was able to enter that data, but not anymore with VD.

**Comment by ezwep** [comment]
I also find that the energy that you enter in the virtual device is not displayed in Homey. It would be nice if I could fill this in myself via the advanced settings. In my case I have a Shelly 4 Pro device in my garage where 3 different lights are connected. I want to be able to operate these as virtual devices in the different zones and to be able to monitor the energy. I wanted to do this with your app; only this does not work.
I also see that more people have experienced the same thing: https://community.athom.com/t/virtual-devices-energy-consumption/20993/6
I prefer to use your app because it is much more complete.

**Comment by ezwep** [comment]
Hi Arjan, is it possible to fix this bug?

**Comment by ArjanKranenburg** [comment]
It is in 'test'

**Comment by ArjanKranenburg** [comment]
I believe this is solved. Let me know if it isn't.
